var/spack/repos/builtin/packages/bufr/package.py

```diff
@@ -98,13 +99,3 @@ def setup_run_environment(self, env):
             suffixes += ["8", "d"]
         for suffix in suffixes:
             self._setup_bufr_environment(env, suffix)
-
-    # setup.py is generated by cmake and therefore lives in the
-    # subdirectory "python" of the package build_directory
-    @run_after("install")
-    def install_python_interface(self):
-        prefix = self.prefix
-        if self.spec.satisfies("+python"):
-            with working_dir(os.path.join(self.build_directory, "python")):
-                args = std_pip_args + ["--prefix=" + prefix, "."]
-                pip(*args)
```

Comment:
Is this not needed anymore?

Reply:
No. I actually added this a while ago, and it was a mistake. Long story short: bufr has its own, crooked Python install step that runs as part of the cmake step. That one installed code in the wrong place (becuase it cut off the second digit of the minor python version): lib/python3.1/site-packages. The installer in lines 104-109 then repeated this (and for some reason it worked) and put it in python3.10. I fixed the wrong install dir in bufr itself (see the new patch) and also sent a PR to nceplibs-bufr. Now the bufr cmake step installs everything in python3.10 directly, and the repeated install (which would now be failing because something in spack develop got cleaned up/sanitized/....) doesn't need to run anymore.